### PR TITLE
Port tooling from unfetch

### PR DESCRIPTION
**Pull request by danderu**
Hi!

I'm importing rollup + buble from unfetch to this library, as requested in issue: #2

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+let { FORMAT } = process.env;
+
+export default {
+	useStrict: false,
+	sourceMap: true,
+	entry: 'src/index.js',
+	plugins: [
+		buble(),
+		FORMAT==='cjs' && replace({
```
Comment (developit) [inline rollup.config.js]: we can remove both `replace()` calls - this repo won't work with them.

Comment (danderu) [inline rollup.config.js]: done!

**Comment by developit** [comment]
Amazing! think it's ready for me to test out a merge now? 👍

**Comment by developit** [comment]
Ah - we'll need to switch `src/index.js` to export stuff instead of having that inlined UMD bundle.  I'm fine doing that if you don't want to, let me know 💯

**Comment by danderu** [comment]
I was planning to do that in the next PR :)

**Comment by developit** [comment]
Will re-check now! Love that we're dropping bytes too 💃

File: test/mocha.opts [hunk 2]
```diff
@@ -0,0 +1 @@
+--compilers babel-register
```
Comment (developit) [inline test/mocha.opts]: Can probably just pass this to `mocha` in the `test` npm-script.

**Comment by developit** [comment]
Merged, tweaked, released as 1.0.0

**Pull request merged**